code/modules/power/tesla/energy_ball.dm

```diff
@@ -68,7 +64,10 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 			loc = get_step(src,move_dir)
 
 /obj/singularity/energy_ball/proc/handle_energy()
-	if(energy >= 300)
+	if(energy >= energy_to_raise)
+		energy_to_lower = energy_to_raise - 10
+		energy_to_raise += energy_to_raise
+		
 		energy -= 300
```

Comment:
Pretty sure this should be `energy -= energy_to_raise`, and moved to the top of the `if`.